ui/src/androidTest/java/com/xmartlabs/bigbang/ui/recyclerview/common/Brand.java

```diff
@@ -0,0 +1,14 @@
+package com.xmartlabs.bigbang.ui.recyclerview.common;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import lombok.Builder;
+import lombok.Data;
+
+@Builder
+@Data
+public class Brand {
```

Comment:
I'm guessing this is for using as an example, but please move it to the test package